src/Snackbar/SnackbarBody.js

```diff
@@ -26,20 +26,25 @@ function getStyles(props, context) {
   } = context;
 
   const isSmall = width === SMALL;
+  const isMultiline = props.messageElement && props.messageElement.offsetHeight > 20;
+  const rootHeight = isMultiline ? '80px' : `${desktopSubheaderHeight}px`;
```

Comment:
Hardcoded to `80px`? Won't this just make it support 2 lines and not arbitrarily many lines?